userscript.js

```diff
@@ -54,6 +54,7 @@
 // @match          https://www.linkedin.com/safety/go?url=*
 // @match          https://www.luogu.com.cn/discuss/*
 // @match          https://www.luogu.com.cn/paste/*
+// @match          https://www.luogu.com.cn/article/*
```

Comment:
Can you help add the redirection rule after [line 111](https://github.com/OldPanda/Open-the-F-king-URL-Right-Now/pull/136/files#diff-5cd4037ed217d72c140b8270f967eab08cd73388ad2b42ed0bb22f011afbdbdfR111)? Thank you!